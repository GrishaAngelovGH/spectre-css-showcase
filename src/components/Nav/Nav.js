const Nav = () => {
  return (
    <ul className='nav mx-2'>
      <li className='nav-item'>
        <a href='#'>Elements</a>
      </li>
      <li className='nav-item active'>
        <a href='#'>Layout</a>
        <ul className='nav'>
          {
            ['Flexbox grid', 'Responsive', 'Navbar', 'Empty states'].map((v, i) => (
              <li key={i} className='nav-item'>
                <a href='#'>{v}</a>
              </li>
            ))
          }
        </ul>
      </li>
      <li className='nav-item'>
        <a href='#'>Components</a>
      </li>
      <li className='nav-item'>
        <a href='#'>Utilities</a>
      </li>
    </ul>
  )
}

export default Nav