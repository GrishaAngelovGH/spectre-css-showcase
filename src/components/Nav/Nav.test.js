import { render } from '@testing-library/react'

import Nav from './Nav'

test('should render Nav component', () => {
  const view = render(<Nav />)

  expect(view).toMatchSnapshot()
})