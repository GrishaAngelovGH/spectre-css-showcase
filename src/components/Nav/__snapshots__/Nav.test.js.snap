// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Nav component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="nav mx-2"
      >
        <li
          class="nav-item"
        >
          <a
            href="#"
          >
            Elements
          </a>
        </li>
        <li
          class="nav-item active"
        >
          <a
            href="#"
          >
            Layout
          </a>
          <ul
            class="nav"
          >
            <li
              class="nav-item"
            >
              <a
                href="#"
              >
                Flexbox grid
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                href="#"
              >
                Responsive
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                href="#"
              >
                Navbar
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                href="#"
              >
                Empty states
              </a>
            </li>
          </ul>
        </li>
        <li
          class="nav-item"
        >
          <a
            href="#"
          >
            Components
          </a>
        </li>
        <li
          class="nav-item"
        >
          <a
            href="#"
          >
            Utilities
          </a>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="nav mx-2"
    >
      <li
        class="nav-item"
      >
        <a
          href="#"
        >
          Elements
        </a>
      </li>
      <li
        class="nav-item active"
      >
        <a
          href="#"
        >
          Layout
        </a>
        <ul
          class="nav"
        >
          <li
            class="nav-item"
          >
            <a
              href="#"
            >
              Flexbox grid
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              href="#"
            >
              Responsive
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              href="#"
            >
              Navbar
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              href="#"
            >
              Empty states
            </a>
          </li>
        </ul>
      </li>
      <li
        class="nav-item"
      >
        <a
          href="#"
        >
          Components
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          href="#"
        >
          Utilities
        </a>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
